test/IDE/complete_overlaymodule.swift

```diff
@@ -0,0 +1,37 @@
+// RUN: %target-swift-ide-test -code-completion -source-filename %s -I %S/Inputs/mock-sdk -F %S/Inputs/mock-sdk -code-completion-token=TYPE_GLOBAL | %FileCheck %s --check-prefix=TYPE_GLOBAL
+// RUN: %target-swift-ide-test -code-completion -source-filename %s -I %S/Inputs/mock-sdk -F %S/Inputs/mock-sdk -code-completion-token=EXPR_GLOBAL | %FileCheck %s --check-prefix=EXPR_GLOBAL
+// RUN: %target-swift-ide-test -code-completion -source-filename %s -I %S/Inputs/mock-sdk -F %S/Inputs/mock-sdk -code-completion-token=EXPR_MEMBER | %FileCheck %s --check-prefix=EXPR_MEMBER
+
+import OverlayTest
+
+func testGlobalType() {
+    let _: #^TYPE_GLOBAL^#
+// TYPE_GLOBAL: Begin completions
+// TYPE_GLOBAL-NOT: OverlayTest[#Module#]
+// TYPE_GLOBAL-DAG: Decl[Module]/None:                  OverlayTest[#Module#];
```

Comment:
Yeah, I think you are right. Also, `swift-idea-test` doesn't sort the results. Do you know a better way to check this? Is possible, I want to avoid 2 `FileCheck` invocations for these checks.

Reply:
Outputting to a file and running file check for the negative and positive checks separately is all I can think of, sorry 🙁